plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/pom.xml

```diff
@@ -18,60 +19,22 @@
 				<extensions>true</extensions>
 			</plugin>
 			<plugin>
-				<groupId>com.googlecode.maven-download-plugin</groupId>
-				<artifactId>download-maven-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>fetch-grammar</id>
-						<phase>generate-resources</phase>
-						<goals>
-							<goal>wget</goal>
-						</goals>
-						<configuration>
-							<url>https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/archive/master.zip</url>
-							<unpack>true</unpack>
-							<outputDirectory>server/</outputDirectory>
-							<skip>false</skip>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>exec-maven-plugin</artifactId>
-				<version>1.6.0</version>
+				<groupId>org.codehaus.mojo</groupId>
 				<executions>
 					<execution>
-						<id>npm-install</id>
-						<goals>
-							<goal>exec</goal>
-						</goals>
-						<phase>generate-resources</phase>
-						<configuration>
-							<executable>npm</executable>
-							<arguments>
-								<argument>install</argument>
-							</arguments>
-							<workingDirectory>server/fabric8-analytics-lsp-server-master/</workingDirectory>
-						</configuration>
-					</execution>
-					<execution>
-						<id>npm-build</id>
+						<id>LSP SERVER</id>
+						<phase>generate-sources</phase>
 						<goals>
 							<goal>exec</goal>
 						</goals>
-						<phase>generate-resources</phase>
 						<configuration>
-							<executable>npm</executable>
-							<arguments>
-								<argument>run-script</argument>
-								<argument>build</argument>
-							</arguments>
-							<workingDirectory>server/fabric8-analytics-lsp-server-master/</workingDirectory>
+							<executable>${basedir}/scripts/lsp-server.sh</executable>
 						</configuration>
 					</execution>
 				</executions>
 			</plugin>
+
 			<plugin>
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>tycho-packaging-plugin</artifactId>
```

Comment:
is the server/fabric8-analytics-lsp-server-master/ folder still relevant for being jgit-ignored here? Seems like you're unpacking into ca-lsp-server/ now, not server/